Update authors.yml & adding Dev Plan Changes post

havi opened the pull request:
adding author Ryan Stewart

This blog post has the following features:

- [ ] A GitHub Repository with a polished README
- [ ] A GitHub Repository under the github.com/oktadev account
- [ ] A title that's approved by Dev Advocacy
- [ ] A URL that's approved by Dev Advocacy
- [ ] The content has been ran through Grammarly (https://www.grammarly.com/)
- [ ] Rendered locally and confirmed that no Markdown typos exist

havi committed:
Update authors.yml

adding author Ryan Stewart

havi committed:
Add Ryan Stewart avatar

bdemers commented on _source/_data/authors.yml:
```diff
@@ -1,3 +1,11 @@
+full_name: Ryan Stewart
```
```suggestion
ryan-stewart:
  full_name: Ryan Stewart
```

bdemers committed:
Apply suggestions from code review

bdemers commented on _source/_data/authors.yml:
```diff
@@ -1,3 +1,12 @@
+  web: http://ryanstewart.net
+  github: https://github.com/ryanstewart
+  bio:
+  Director of Product Management for CIAM Developer Platform at @okta
```
```suggestion
  bio: "Director of Product Management for CIAM Developer Platform at @okta"
```

bdemers committed:
Apply suggestions from code review

bdemers committed:
remove boarder around ryan's profile image

havi committed:
Add files via upload

Re-using the Auth0-Okta join forces img as social img for dev-plan-changes-post

havi committed:
adding social img

havi committed:
Adding Dev Plan post .md file

havi committed:
Front matter punctuation fix

havi committed:
front matter fix #2

havi committed:
Added dev forum link

mraible committed:
Merge branch 'main' into havi-authors-update

mraible commented on _source/_posts/2021-10-05-okta-dev-plan-changes.md:
```diff
@@ -0,0 +1,31 @@
+author: ryan-stewart 
+by: internal-contributor
+communities: [.net,java,javascript,go,php,python,ruby,devops,announcement]
+description: "This announces how we are streamlining and clarifying our developer product offering in light of the Auth0 acquisition. We are sunsetting Okta's Developer Advanced plan, and providing Auth0 plans for developers seeking a fully self-service experience."
```
This description might be a bit lengthy. It's recommended descriptions are under 150 characters. https://moz.com/learn/seo/meta-description

havi replied:
```suggestion
description: "In light of the Auth0 acquisition, we are streamlining Okta's developer product offering by retiring Okta's Developer Advanced plan, and providing Auth0 plans for developers seeking a fully self-service experience."
```

havi committed:
Update _source/_posts/2021-10-05-okta-dev-plan-changes.md

reducing length of description

bdemers committed:
Merge branch 'main' into havi-authors-update

bdemers committed:
Fixing curly quotes

bdemers committed:
add new announcement category
